### PR TITLE
Blob header

### PR DESCRIPTION
**Pull request by nugaon**
Defining the first 4 bytes of all FDP Blobs.

### CODE REVIEW

File: text/0056-blob-header.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# Summary
+This document defines the structure of the first 4 bytes of all FDP Blobs.
+
+# Context and motivation
+There should be an exact and simple differentiation between the introduced data-structures.
+
+All data-structures have byte-representation in order to be saved on storage or transmitted over the network. 
+This can be later used for initiating the in-memory object again (serialization/deserialization).
+
+Facilitating the work of functions dealing with FDP data-structures, the data-blobs first 4 bytes have to be verified only for identifying the _type_ and _version_ of the data-blob (e.g. the data-blob in question is a BeeSon with version 0.1.0).
```
Comment (crtahlin) [inline text/0056-blob-header.md:16]: the data-blobs first 4 bytes ---> the first 4 bytes of data-blobs

File: text/0056-blob-header.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+This first 4 bytes of a FDP Blob is called FDP Blob Header.
+
+# Guide-level explanation
+Every FDP Data Blob type has a reserved identifier that is defined in the first byte of FDP data blobs.
```
Comment (crtahlin) [inline text/0056-blob-header.md:21]: use consistent wording - either "data blobs" or "data-blobs"

File: text/0056-blob-header.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+# Reference-level explanation
+You can submit any data-structure which has at least a JavaScript library to serialize/deserialize it and additionally it:
+- does not violate FDP principles (which are written in FIP documents like this proposal)
+- offers a significant feature that is not possible to reproduce or to effectively substitue with the currently active and used FDP Data Blobs
+
+For submitting a data-structure, create a PR that extends the FDP Blob Type table above and link the open-source repository of the JS library.
+The identifier can be chosen freely from the available set, ranging from 0 to 255 expect for reserved ones but that must be accepted by the PR reviewers.
+
+# Drawbacks
+The range for the FDP data-structures may seen short, but it is designed to prevent the FDP stack from increasing in complexity (see details in the section below).
```
Comment (crtahlin) [inline text/0056-blob-header.md:43]: seen ---> seem

**Review by crtahlin — COMMENTED** [review]
Left single line comments.

**Review by molekilla — APPROVED** [review]
our own multiformats tables.csv!!!

**Pull request merged**